src/App.js

```diff
@@ -1,9 +1,19 @@
-import React from 'react'
+import React from "react";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+
+import Search from "./components/Search";
+import UserProfile from "./components/UserProfile";
+import ErrorPage from "./components/ErrorPage";
 
 export default function App() {
-    return (
-        <div>
-            Hello Parcel
-        </div>
-    )
+  return (
+    <Router>
+      <Switch>
+        <Route exact path="/" component={Search}></Route>
```

Comment:
Can you do `<Route exact path="/" component={Search} />` for components who have no children?